data/iso_iec_dir_1_iso_sup_ed-11-2020.yml

```diff
@@ -0,0 +1,59 @@
+title:
+- type: title-intro
+  content: ISO/IEC Directives
+  language: en
+  script: Latn
+  format: text/plain
+- type: title-main
+  content: Consolidated ISO Supplement
+  language: en
+  script: Latn
+- type: title-part
+  content: Procedures specific to ISO
+  language: en
+  script: Latn
+- type: title-intro
+  content: Directives ISO/IEC
+  language: fr
+  script: Latn
+  format: text/plain
+- type: title-main
+  content: Supplément ISO consolidé
+  language: fr
+  script: Latn
+- type: title-part
+  content: Procédures spécifiques à l'ISO
+  language: fr
+  script: Latn
+
+part: "1"
+
+link:
+- type: src
+  content: https://fdocuments.in/document/isoiec-directives-part-1-consolidated-iso-supplement-and-iso-proceduresiso.html
```

Comment:
@anermina let's remove all non-authoritative sources (the only authoritative source is iso.org). Rather have them empty.